core/src/bluetooth.py

```diff
@@ -0,0 +1,20 @@
+from trezorio import BLE, ble
```

Comment:
One more thing, can you please
* move the file under `core/src/trezor/` &
* update `SConscript.*` to only include it if BLE is available?

Or we can use the C/Rust version if you have it. I'd like to merge this soon since it doesn't look like it can break USB-only models and the number of BLE-related branches is getting a  bit out of hand (my bad).

Reply:
Replaced `bluetooth.py` by C extension in 3faf767d0.